spec/graphql/list_type_spec.rb

```diff
@@ -8,6 +8,10 @@
     assert_equal([1.0, 2.0, 3.0].inspect, float_list.coerce_isolated_input([1, 2, 3]).inspect)
   end
 
+  it "converts items that are not lists into lists" do
+    assert_equal([1.0].inspect, float_list.coerce_isolated_input(1.0).inspect)
```

Comment:
Why the use of `.inspect` here (and below)? Did the test fail without it? 

If it works without it, I'd rather remove it, that way we can be sure we're testing the return value of this method, not the return value's `#inspect` method.